crates/atuin-client/src/database.rs

```diff
@@ -118,42 +118,59 @@ pub trait Database: Send + Sync + 'static {
     async fn stats(&self, h: &History) -> Result<HistoryStats>;
 }
 
-// Intended for use on a developer machine and not a sync server.
-// TODO: implement IntoIterator
-#[derive(Debug, Clone)]
-pub struct Sqlite {
-    pub pool: SqlitePool,
-}
+macro_rules! create_sqlite_pool {
```

Comment:
Generally happy with this, thank you! Would it be possible to drop the macro though please?

Seeing as it's only used in two places, I don't think it is necessary. Generally try to keep things simple unless the benefit is really worth it!